app/views/job_templates/cloud_connector.erb

```diff
@@ -33,4 +33,4 @@ feature: ansible_configure_cloud_connector
   vars:
     satellite_cloud_connector_url: "<%= foreman_server_url %>"
```

Comment:
you'll also have to update this to `foreman_cloud_connector_url`

see the readme at https://github.com/theforeman/foreman-operations-collection/tree/develop/roles/cloud_connector